apps/swirl-docs/src/components/Layout/CategoryNav.tsx

```diff
@@ -51,7 +51,7 @@ export const CategoryNav: FunctionComponent<CategoryNavProps> = ({
   return (
     <nav
       aria-label="category"
-      className="hidden md:block px-4 border-r-1 w-80 min-w-[20rem] max-w-xs"
+      className="hidden lg:block px-4 border-r-1 w-80 min-w-[20rem] max-w-xs"
     >
       <ul className="mt-6">
         {categoryLinkList?.map((navItem: NavItem, index) => {
```

Comment:
**issue:** You are using `<NoSsr>` inside the SubElement which results in invalid markup for the list.

```
<ul>
  <div>
    <li>…</li>
  </div>
  …
</ul>